Edit javascript codebyte1 (#2245)
Codecademy/docs: content/javascript/concepts/dates/terms/getDay/getDay.md

  - 'paths/front-end-engineer-career-path'
---

The **`.getDay()`** method returns the day of the week.

## Syntax

> ```js

Reviewer: ```suggestion
```shell
```

myDate.getDay();
```

> The return value of `getDay()` is an integer number between `0` and `6`:

Reviewer: ```suggestion
The return value of `.getDay()` is an integer number between `0` and `6`:
```

// Output: It is Friday, let's have a coffee later!
```

## Codebyte Example

> The example below is runnable, and first, it defines a variable called `today` with a new `Date` of today's date, then uses the `getDay()` method to store the weekday from that date into a variable called `weekDay`. Then `console.log` prints the number representing that day of the week to the screen:

Reviewer: ```suggestion
The example below is runnable, and first, it defines `today` with today's date, then the result of the `.getDay()` method is assigned to `weekDay`. Finally, a `console.log` call prints the value to the screen:
```
